ix-dev/community/prowlarr/questions.yaml

```diff
@@ -0,0 +1,340 @@
+groups:
+  - name: Prowlarr Configuration
+    description: Configure Prowlarr
+  - name: User and Group Configuration
+    description: Configure User and Group for Prowlarr
+  - name: Advanced Pod Configuration
+    description: Configure Advanced Pod Options for Prowlarr
+  - name: Network Configuration
+    description: Configure Network for Prowlarr
+  - name: Storage Configuration
+    description: Configure Storage for Prowlarr
+  - name: Resources Configuration
+    description: Configure Resources for Prowlarr
+
+questions:
+  - variable: prowlarr
+    label: ""
+    group: Prowlarr Configuration
+    schema:
+      type: dict
+      attrs:
+        - variable: instance_name
+          label: Instance Name
+          description: The name of the Prowlarr instance.
+          schema:
+            type: string
+            default: "Prowlarr"
+            required: true
+        - variable: additional_envs
+          label: Additional Environment Variables
+          description: Configure additional environment variables for Prowlarr.
+          schema:
+            type: list
+            default: []
+            items:
+              - variable: env
+                label: Environment Variable
+                schema:
+                  type: dict
+                  attrs:
+                    - variable: name
+                      label: Name
+                      schema:
+                        type: string
+                        required: true
+                    - variable: value
+                      label: Value
+                      schema:
+                        type: string
+                        required: true
+  - variable: run_as
+    label: ""
+    group: User and Group Configuration
+    schema:
+      type: dict
+      attrs:
+        - variable: user
+          label: User ID
+          description: The user id that Prowlarr files will be owned by.
+          schema:
+            type: int
+            min: 568
+            default: 568
+            required: true
+        - variable: group
+          label: Group ID
+          description: The group id that Prowlarr files will be owned by.
+          schema:
+            type: int
+            min: 568
+            default: 568
+            required: true
+
+  - variable: network
+    label: ""
+    group: Network Configuration
+    schema:
+      type: dict
+      attrs:
+        - variable: web_port
+          label: WebUI Port
+          description: The port for Prowlarr WebUI
+          schema:
+            type: int
+            default: 30027
+            min: 9000 # TODO: do we still need this?
```

Comment:
I would assume 9000 is no longer a restriction. We should probably lift it and only restrict specific ports we already know are in use by TrueNAS out of box. I.E. 443/80, etc.

Reply:
Yea that's what I thought, I should probably update the type here and use the new `port` $ref, that will filter used ports too.